contracts/VestingEscrowSimple.vy

```diff
@@ -193,12 +186,14 @@ def renounce_ownership():
     @notice Renounce admin control of the escrow
     """
     assert msg.sender == self.admin  # dev: admin only
-    self.future_admin = ZERO_ADDRESS
-    self.admin = ZERO_ADDRESS
-    log ApplyOwnership(ZERO_ADDRESS)
+    self.future_admin = empty(address)
+    self.admin = empty(address)
+    log ApplyOwnership(empty(address))
+
 
 @external
-def collect_dust(token: address):
-    assert msg.sender == self.recipient  # dev: recipient only
-    assert (token != self.token.address or block.timestamp > self.disabled_at)
-    assert ERC20(token).transfer(self.recipient, ERC20(token).balanceOf(self))
+def collect_dust(token: address, beneficiary: address = msg.sender):
```

Comment:
`token: address` can be changed to `token: ERC20`, then you won't have to wrap it in the interface every time

add dev comment